base/mpfr.jl

```diff
@@ -26,6 +26,9 @@ import Base.Math.lgamma_r
 
 import Base.FastMath.sincos_fast
 
+const MPFR_VERSION = VersionNumber(unsafe_string(ccall((:mpfr_get_version,:libmpfr), Ptr{Cchar}, ())))
+const MPFR_PATCHES = split(unsafe_string(ccall((:mpfr_get_patches,:libmpfr), Ptr{Cchar}, ())),' ')
```

Comment:
Okay, I've changed it to a function for now. We should think about how to define a consistent interface for library versions (both for Base and in packages).